config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
bu dosya neden geldi?

Reply:
Abi bunlar rails 6.1 ile geldi. rails app:update dediğin zaman geliyor. Framework'e eklenen yeni featureler ile gelen configleri saklıyor.